ios/iden3CoreSDK/Iden3Error.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Iden3Error.swift
+//  iden3CoreSDK
+//
+//  Created by Iden3 on 12/06/2020.
```

Comment:
We don't use this license. The license for the code can be found in the root of the repo: `/LICENSE`.

Reply:
done